test/Microsoft.AspNet.Mvc.Core.Test/ParameterBinding/ModelBindingHelperTest.cs

```diff
@@ -23,10 +23,7 @@ public class ModelBindingHelperTest
         public async Task TryUpdateModel_ReturnsFalse_IfBinderReturnsFalse()
         {
             // Arrange
-            var metadataProvider = new Mock<IModelMetadataProvider>();
-            metadataProvider.Setup(m => m.GetMetadataForType(It.IsAny<Type>()))
-                            .Returns(new ModelMetadata(metadataProvider.Object, null, typeof(MyModel), null))
-                            .Verifiable();
+            var metadataProvider = new EmptyModelMetadataProvider();
```

Comment:
Use `EmptyModelMetadataProvider` if you are testing model metadata - use `TestModelMetadataProvider.CreateDefaultProvider()` if you are testing model binding or display, and want something that resembles the default configuration.